### ISSUE TITLE
toSVG ignores {config:background}

### ISSUE THREAD

**Issue by danyelf** [vega/vega#1037]
I have added a background color to my Vega-lite specification, which looks fine in the editor, but the background color doesn't render when I run toSVG on it. This is important because I want to print against a non-white background, and so want to set a background color.

This zip file contains the SVG
[singleview_11_bar-chart.zip](https://github.com/vega/vega/files/1381084/singleview_11_bar-chart.zip)

```
{

  "$schema": "https://vega.github.io/schema/vega-lite/v2.json",
  "config": {"background":"#fdf5e6"},    
  "data": { "url": "data/cars.csv" },
  "mark": "bar",
  "encoding": {
    "x": {"field": "body-style", "type": "nominal",  "axis": {"title": "Body Style"}},
    "y": {"aggregate": "mean", "field": "curb-weight", "type": "quantitative", "axis": {"title": "Curb Weight"}}
  }
}
```

**Comment by jheer** [comment]
Thanks for filing the issue! One hiccup here is that the SVG spec does not support a top-level background attribute. The _browser_ does via HTML CSS, but there is no guarantee of support in other tools (e.g., Inkscape, Illustrator). HTML CSS is what Vega's SVGRenderer uses, but our SVGStringRenderer (used for rapid SVG export) currently does not.

A globally-acceptable alternative may be to instead generate an internal SVG `rect` element with the background color, using the same dimensions as the SVG element. I think we will pursue this direction, please feel free to share additional thoughts.

**Comment by jheer** [comment]
Fixed by changes in [vega-scenegraph v2.0.4](https://github.com/vega/vega-scenegraph/releases/tag/v2.0.4). SVG export (from either renderer) now includes a background `rect` element if the background color has been defined. Browser-based SVG stills uses CSS styling. Will be included in the next Vega release.
